firmware/src/sys/board.hpp

```diff
@@ -58,6 +58,9 @@ void resetWatchdog();
 
 void setStatusLed(bool state);
 void setCanLed(bool state);
+void DutycycleCounterPP();
+void DutycycleCounterMM();
+signed DutycycleCounterRead();
```

Comment:
void DutycycleCounterPP(); 
This one increase the DutycycleCounter, called every 100ms from main
DutycycleCounterMM();
And this one decreased DutycycleCounter every time the charge pump runs, called from charger
signed DutycycleCounterRead();
returns the DutycycleCounter value, used by magnet. negative value = don't run. positive = run